udl-list.md

```diff
@@ -303,3 +303,5 @@
 | [Excel formulas](./UDLs/ExcelFormula_byOlivierMarche.xml) | Excel formulas | [Olivier Marché](https://github.com/OlivierMarche)
 | [Excel formulas (Dark)](./UDLs/ExcelFormula_ThemeDark_byOlivierMarche.xml) | Excel formulas (Dark theme) | [Olivier Marché](https://github.com/OlivierMarche)
 | [AMXX 1.8.3](./UDLs/AMXX_1.8.3_OciXCrom.xml) | AMXX 1.8.3 | [OciXCrom](https://github.com/OciXCrom)
+| [Amiga_E](./UDLs/Amiga_E_bydmcoles.xml) | Amiga E | dmcoles(https://github.com/dmcoles) |
```

Comment:
put the `[` `]` around `dmcoles`, so change

> `dmcoles(https://github.com/dmcoles)`

to

> `[dmcoles](https://github.com/dmcoles)`

so that the link will be correct.

After this change, I think I'll be able to merge the PR.